packages/manager/modules/web-paas/src/add/add.html

```diff
@@ -137,61 +137,82 @@
+                    option="addon"
+                    plan="$ctrl.selectedPlan"
                 >
-                    <oui-select
-                        disabled="$ctrl.project.configuration.storage && $ctrl.availableStorages.length === 1"
-                        items="$ctrl.availableStorages"
-                        match="name"
-                        model="$ctrl.project.configuration.storage"
-                        name="project-storage"
-                        required
+                </web-paas-additional-option>
+            </oui-step-form>
+            <oui-step-form
+                data-header="{{:: 'Summary' | translate }}"
+                data-submit-text="{{:: 'web_paas_add_project_next' | translate }}"
+                data-loading="$ctrl.isGettingCheckoutInfo"
+                data-on-submit="$ctrl.platformOrder()"
+            >
+                <div>
+                    <h5
+                        data-translate="web_paas_add_project_license_summary_offer"
+                    ></h5>
+                    <span data-ng-bind="$ctrl.selectedPlan.invoiceName"></span>
+                    <div
+                        data-ng-repeat="addon in $ctrl.selectedPlan.addons track by $index"
                     >
-                        <span data-ng-bind="$item.name"></span>
-                    </oui-select>
-                </oui-field>
-                <oui-field
-                    data-label="{{:: 'web_paas_add_project_options_environment' | translate}}"
-                    data-size="xl"
+                        <span
+                            data-ng-if="addon.family === 'staging_environment'"
+                            >+<span data-ng-bind="addon.quantity"></span>
+                            <span
+                                data-ng-bind="$ctrl.selectedPlan.vcpuConfig"
+                            ></span>
+                        </span>
+                        <span
+                            data-ng-if="addon.family === 'storage'"
+                            data-translate="web_paas_add_project_license_summary_addon_storage"
+                            data-translate-values="{quantity: addon.quantity}"
+                        ></span>
+                        <span
+                            data-ng-if="addon.family === 'user_license'"
+                            data-translate="web_paas_add_project_license_summary_addon_user_licence"
+                            data-translate-values="{quantity: addon.quantity}"
+                        ></span>
+                    </div>
+                </div>
+                <div class="mt-3">
+                    <h5
+                        data-translate="web_paas_add_project_license_summary_total"
+                    ></h5>
+                    <div>
+                        <dl data-class="col-md-2">
+                            <dt
+                                data-translate="web_paas_add_project_license_total_price_without_tax"
```

Comment:
I am still waiting for the price content from PU for this. As we have to show prorata price and next months price here, this can stay here temporarily

Reply:
Has been removed and have used `<ovh-manager-catalog-price>` now.